Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
+
+- Bump MongoDB.Driver from 2.13.2 to 2.14.0 and Microsoft.AspNetCore.Http from 2.1.0 to 2.2.0. [#608](https://github.com/microsoft/sarif-pattern-matcher/pull/608)
```

Comment:
>14 [](http://example.com/codeflow?start=39&length=2)

version is wrong

Reply:
fixed.
I mistakenly used the version of ElfSharp (it was in my head).